### PR TITLE
SAL-3997_add_documentupdate

### PR DESCRIPTION
**Pull request by AlexanderTereshchenko**

### CODE REVIEW

File: src/Services/DocumentsService.php [hunk 1]
```diff
@@ -40,6 +41,24 @@ public function create(CreateModel $document): Document
...
     }
 
+    /**
+     * Update document
+     *
+     * @param UpdateModel $document
+     * @return DocumentEntity
+     * @throws \Exception
+     */
+    public function update(UpdateModel $document)
```
Comment (stanislavkosatkinpf) [inline src/Services/DocumentsService.php]: Return type

File: src/Models/Document/Update.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+/**
+ * Class Update
+ * @package AirSlate\ApiClient\Models\Document
+ */
+class Update extends Create
+{
+    public $id;
+
+    public static function createFromDocument(Document $document, array $data = []): Update
```
Comment (stanislavkosatkinpf) [inline src/Models/Document/Update.php]: PHPDoc

File: src/Services/DocumentsService.php (hunk 1 above)
Comment (AlexanderTereshchenko) [reply src/Services/DocumentsService.php]: +

File: src/Models/Document/Update.php (hunk 2 above)
Comment (AlexanderTereshchenko) [reply src/Models/Document/Update.php]: +

**Pull request merged**